content/blog/reprex-2-0-0/index.Rmd

```diff
@@ -0,0 +1,153 @@
+---
+output: hugodown::hugo_document
+
+slug: reprex-2-0-0
+title: reprex 2.0.0
+date: 2021-04-02
+author: Jenny Bryan
+description: >
+    reprex is now much easier to use on RStudio Server and RStudio Cloud.
+photo:
+  url: https://unsplash.com/photos/NROHA1B-NYk
+  author: Mitchell Luo
+categories: [package] 
+---
+
+We're overjoyed to announce the release of [reprex](https://reprex.tidyverse.org) 2.0.0.
+reprex is a package that helps you prepare **REPR**oducible **EX**amples to share in places where people talk about code, e.g., on GitHub, on Stack Overflow, and in Slack or email messages.
+
+You can install the current version of reprex from CRAN with[^1]:
+
+[^1]: Another way you might get reprex is by installing the tidyverse meta-package.
+reprex is one of the packages installed by `install.packages("tidyverse")`, however it is **not** among the core packages attached by `library(tidyverse)`.
```

Comment:
```suggestion
reprex is one of the packages installed by `install.packages("tidyverse")`, however it is **not** among the [core packages](https://www.tidyverse.org/packages/#core-tidyverse) attached by `library(tidyverse)`.
```

Reply:
Feel free to reject, just thought it might be nice, since it sort of begs the question, _Which packages are core?_